src/metaswitch/crest/settings.py

```diff
@@ -100,7 +100,7 @@ def ensure_dir_exists(d):
 CYCLONE_DEBUG = False  # Make cyclone emit debug messages to the browser etc.
 
 # Include any locally-defined settings.
-_local_settings_file = os.path.join(_MY_DIR, "local_settings.py")
+_local_settings_file = os.environ.get('CREST_SETTINGS', os.path.join(_MY_DIR, "local_settings.py"))
```

Comment:
I tried that at first, however settings are loaded at import time (as settings are attributes of the `settings` module), which is before command-line arguments are parsed.  There are a few import-time users of settings, too, which means we can't delay loading of the settings file.